### PR TITLE
W-14677283 update spec duke

### PR DESCRIPTION
**Pull request by dukesphere**

### CODE REVIEW

File: modules/ROOT/pages/imp-implement-apis.adoc [hunk 1]
```diff
@@ -188,11 +168,44 @@ See the xref:tut-graphql-implement-api.adoc[] tutorial for an example GraphQL AP
...
+--
+<dependency>
+    <groupId>e91cab06-650b-4634-9c6f-5bc4f4fc4d17</groupId>
+    <artifactId>phoneSync1005</artifactId>
+    <version>2.0.0</version>
+    <classifier>oas</classifier>
+    <type>zip</type>
+</dependency>
+--
+** Updates the configuration XML file that contains the interface for your implementation project with the changes to the imported specification.
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-implement-apis.adoc]:
```suggestion
** Updates the configuration XML file that contains the interface for your project with the changes to the imported specification.
```

File: modules/ROOT/pages/imp-implement-apis.adoc [hunk 2]
```diff
@@ -17,36 +15,31 @@ autogenerate a basic interface in your implementation project.
...
 which you then implement within a Mule application.
 
-
-// * Interface: receives all incoming requests to your application.
-// * Scaffolded flows in the interface: validate and route requests.
-// * Implementation: provides the backend logic for calls to the interface.
-
 The process you use to implement your API depends on whether you already
-created an integration project or published the spec to Exchange:
+created an integration or implementation project, or published the spec to Exchange:
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-implement-apis.adoc]:
```suggestion
created a project for your integration or implementation, or published the spec to Exchange:
```

File: modules/ROOT/pages/imp-implement-apis.adoc [hunk 3]
```diff
@@ -17,36 +15,31 @@ autogenerate a basic interface in your implementation project.
...
 * Optionally, xref:int-create-integrations.adoc[create your integration project (Mule app)].
 
 
 [[scaffold-new-integration]]
-== Scaffold an API and Create an Integration Project
+== Scaffold an API into a New Project
 
-If you have not already created an integration project, use Anypoint Code Builder to
-scaffold the API in a new Mule project:
+If you have not already created an implementation project, use Anypoint Code Builder to scaffold the API in a new Mule project:
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-implement-apis.adoc]:
```suggestion
If you have not already created a project, use Anypoint Code Builder to scaffold the API in a new Mule project:
```

File: modules/ROOT/pages/imp-implement-apis.adoc [hunk 4]
```diff
@@ -102,9 +95,9 @@ When you create the project, Anypoint Code Builder:
 </dependency>
 --
 +
-** Scaffolds your API into the new integration project and opens the configuration XML file.
+** Scaffolds your API into the new implementation project and opens the configuration XML file.
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-implement-apis.adoc]:
```suggestion
** Scaffolds your API into the new Mule project and opens the configuration XML file.
```

File: modules/ROOT/pages/imp-implement-apis.adoc [hunk 5]
```diff
@@ -102,9 +95,9 @@ When you create the project, Anypoint Code Builder:
...
 +
-The configuration XML file includes the interface for your integration project,
+The configuration XML file includes the interface for your implementation project,
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-implement-apis.adoc:100]:
```suggestion
The configuration XML file includes the interface for your project,
```

File: modules/ROOT/pages/imp-implement-apis.adoc [hunk 6]
```diff
@@ -188,11 +168,44 @@ See the xref:tut-graphql-implement-api.adoc[] tutorial for an example GraphQL AP
...
+
+Re-scaffolding enables you to update a project in the IDE with the latest changes and version of an API spec in Anypoint Exchange. Re-scaffolding updates the configuration XML and `pom.xml` in your project. To re-scaffold a spec defined in a local API design project that is not published to Exchange, see xref:imp-implement-local-apis.adoc[], instead.
+
+This procedure assumes that an updated version of the spec is available from Exchange, for example, a 2.0.0 version:
 
+image::imp-assets-from-exchange-multiversion.png["Selecting a new version of an API spec from Exchange"]
 
-// add relevant stuff from tut-implement-am-flights-api
-// Then, add the logic to implement an endpoint in the API.
+. In Anypoint Code Builder, open your implementation project in the Explorer view, for example, `my-implementation-project`.
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-implement-apis.adoc:180]:
```suggestion
. In Anypoint Code Builder, open your project in the Explorer view, for example, `my-implementation-project`.
```

File: modules/ROOT/pages/imp-implement-apis.adoc [hunk 7]
```diff
@@ -188,11 +168,44 @@ See the xref:tut-graphql-implement-api.adoc[] tutorial for an example GraphQL AP
...
+<dependency>
+    <groupId>e91cab06-650b-4634-9c6f-5bc4f4fc4d17</groupId>
+    <artifactId>phoneSync1005</artifactId>
+    <version>2.0.0</version>
+    <classifier>oas</classifier>
+    <type>zip</type>
+</dependency>
+--
+** Updates the configuration XML file that contains the interface for your project with the changes to the imported specification.
+. Check for the changes to your implementation project.
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-implement-apis.adoc:205]:
```suggestion
. Check for the changes to your project.
```

File: modules/ROOT/pages/imp-implement-apis.adoc [hunk 8]
```diff
@@ -188,11 +168,44 @@ See the xref:tut-graphql-implement-api.adoc[] tutorial for an example GraphQL AP
...
++
+For example, if the updated specification adds a new endpoint, the configuration XML for the implementation project has a new flow for the endpoint.
```
Comment (dukesphere) [inline modules/ROOT/pages/imp-implement-apis.adoc]:
```suggestion
For example, if the updated specification adds a new endpoint, the configuration XML for the project has a new flow for the endpoint. 
```

**Review by dukesphere — COMMENTED** [review]
removing "implementation" from "implementation project" where context is clear

File: modules/ROOT/pages/imp-implement-apis.adoc [hunk 9]
```diff
@@ -74,10 +67,10 @@ include::partial$acb-reusable-steps.adoc[tags="proj-directory-warn"]
 | *Show filters* |See xref:filter-search-results.adoc[] for more information.
 |===
 
-. Search for an API specification on Exchange:
+. Search for an API spec on Exchange:
 +
 .. Start typing the name of an API specification.
```
Comment (marinarv) [inline modules/ROOT/pages/imp-implement-apis.adoc]: should we always use spec instead of specification?

Comment (dukesphere) [reply modules/ROOT/pages/imp-implement-apis.adoc]: We've started using "spec" these days ¯\_(ツ)_/¯ It's shorter and well understood.

Comment (marinarv) [reply modules/ROOT/pages/imp-implement-apis.adoc]: exactly, wondering if also affects `Start typing the name of an API specification.`

**Review by marinarv — APPROVED** [review]

File: modules/ROOT/pages/imp-implement-apis.adoc (hunk 9 above)
Comment (dukesphere) [reply modules/ROOT/pages/imp-implement-apis.adoc]: ohhh, thx for catching that

Comment (dukesphere) [reply modules/ROOT/pages/imp-implement-apis.adoc]: conformed to new spelling everywhere.

**Review by chris-campbell9 — APPROVED** [review]

**Pull request merged**